maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/mapper/ApplicationTrackingMapper.java

```diff
@@ -57,27 +63,47 @@ protected Ioj buildIOJ(WorkflowRequest workflowRequest) {
                 .withIojResult(application.getIojResult())
                 .withIojReason(application.getIojResultNote())
                 .withIojAppealResult(ReviewResult.getFrom(iojAppealDTO.getAppealDecisionResult()))
+                .withIojAssessorName(getCapitalisedFullName(repOrderDTO.getUserCreatedEntity().getFirstName(), repOrderDTO.getUserCreatedEntity().getSurname()))
                 .withAppCreatedDate(toLocalDateTime(application.getDateCreated()));
 
     }
 
-    protected Passport buildPassport(WorkflowRequest workflowRequest) {
+    protected Passport buildPassport(WorkflowRequest workflowRequest, RepOrderDTO repOrderDTO) {
 
         ApplicationDTO application = workflowRequest.getApplicationDTO();
         PassportedDTO passportedDTO = application.getPassportedDTO();
+        PassportAssessmentDTO passportAssessmentDTO = null;
+        if (passportedDTO.getPassportedId() != null) {
+            passportAssessmentDTO = repOrderDTO.getPassportAssessments()
+                    .stream()
+                    .filter(pa -> pa.getId() == passportedDTO.getPassportedId().intValue())
+                    .findFirst()
+                    .orElse(null);
+        }
+
         return new Passport()
                 .withPassportId(passportedDTO.getPassportedId() != null ? passportedDTO.getPassportedId().intValue() : null)
                 .withPassportResult(PassportAssessmentResult.getFrom(passportedDTO.getResult()))
+                .withPassportAssessorName((null != passportAssessmentDTO) ?
+                        getCapitalisedFullName(passportAssessmentDTO.getUserCreatedEntity().getFirstName(),
+                                passportAssessmentDTO.getUserCreatedEntity().getSurname()) : null)
                 .withPassportCreatedDate(passportedDTO.getDate() != null ? toLocalDateTime(passportedDTO.getDate()) : null);
     }
 
-    protected MeansAssessment buildMeansAssessment(WorkflowRequest workflowRequest) {
+    protected MeansAssessment buildMeansAssessment(WorkflowRequest workflowRequest, RepOrderDTO repOrderDTO) {
         ApplicationDTO application = workflowRequest.getApplicationDTO();
         FinancialAssessmentDTO financialAssessmentDTO = application.getAssessmentDTO().getFinancialAssessmentDTO();
```

Comment:
To differentiate, we can add a prefix to financialAssementDTO variable, say 'applicationFinancialAssementDTO' and 'maatAPIFinancialAssesmentDTO'

Reply:
Agree..This has been a bit confusing. This is an existing issue, so will create a story to refactor this